src/SFA.DAS.EmployerIncentives.Web/Filters/GoogleAnalyticsFilterAttribute.cs

```diff
@@ -22,22 +26,53 @@ public override void OnActionExecuting(ActionExecutingContext context)
 
         private GoogleAnalyticsData PopulateForEmployer(ActionExecutingContext context)
         {
+            object accountIdRouteValue;
+            object applicationIdRouteValue;
+            
             string hashedAccountId = null;
+            string hashedAccountLegalEntityId = null;
 
             var userId = context.HttpContext.User.Claims.FirstOrDefault(c => c.Type.Equals(EmployerClaims.IdamsUserIdClaimTypeIdentifier))?.Value;
 
-            if (context.RouteData.Values.TryGetValue("employerAccountId", out var employerAccountId))
+            if (context.RouteData.Values.TryGetValue("accountId", out accountIdRouteValue))
```

Comment:
You could get this out of the claims rather than route data.

Reply:
Is that EmployerClaimTypes.Account ? Hashed version of the id?